src/appmixer/everart/lib.generated.js

```
@@ -0,0 +1,116 @@
+    getProperty(obj, path) {
+        return path.split('.').reduce((acc, part) => acc && acc[part], obj);
+    },
+
+    getOutputPortOptions(context, outputType, itemSchema, { label, value }) {
+
+        if (outputType === 'object' || outputType === 'first') {
+            const options = Object.keys(itemSchema)
+                .reduce((res, field) => {
+                    const schema = itemSchema[field];
+                    const label = schema.title;
+                    delete schema.title;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)